specification/specs/common_abstractions/union/Union.ts

```diff
@@ -1,3 +1,12 @@
 @class_serializer("UnionFormatter`2")
-class Union<TFirst, TSecond> {
-}
+class Union<
+  T1 = void,
+  T2 = void,
+  T3 = void,
+  T4 = void,
+  T5 = void,
+  T7 = void,
+  T8 = void,
+  T9 = void,
+  T10 = void
+> {}
```

Comment:
@Mpdreamz this change will make the code more readable, for example:
```diff
- _source: Union<Union<boolean, Field>, Union<Field[], SourceFilter>>;
+ _source: Union<boolean, Field, Field[], SourceFilter>;
```

It's ok for you or it can cause problems to the .NET generator?